apps/pearl-api/utils/web3auth.ts

```diff
@@ -0,0 +1,56 @@
+import { EvmDisplayName } from './chain';
+
+export const CHAIN_CONFIGS: Record<
+  string,
+  {
+    chainNamespace: 'eip155';
+    chainId: string;
+    rpcTarget: string;
+    displayName: string;
+    ticker: string;
+    tickerName: string;
+    blockExplorerUrl: string;
+    logo: string;
+  }
```

Comment:
can we use CustomChainConfig? `import { CustomChainConfig } from '@web3auth/modal';`